### PR TITLE
changed string comparison in ATLUserMock to use NSCaseInsensitiveSearch ...

### PR DESCRIPTION
**Pull request by kmahal**
...for rangeOfString

### CODE REVIEW

**Comment by kcoleman731** [comment]
Where is the test?

**Comment by kmahal** [comment]
@kcoleman731 tests added

File: Tests/ATLUserMockTest.m [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    [super setUp];
+
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
+-(void)testToVerifyCaseInsensitiveSearch{
```
Comment (kcoleman731) [inline Tests/ATLUserMockTest.m]: Format the we use here. 
```
- (void)testToVerifyCaseInsensitiveSearch
{

}
```

Add a space after the dash and then put the opening brace on the next line.

File: Tests/ATLUserMockTest.m [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    NSSet *set2 = [ATLUserMock participantsWithText:@"kleme"];
+    NSString *fullName2 = ((ATLUserMock*)set2.allObjects.firstObject).fullName;
+    expect(fullName2).to.equal([ATLUserMock userWithMockUserName:ATLMockUserNameKlemen].fullName);
+
+    NSSet *set3 = [ATLUserMock participantsWithText:@"bob"];
+    NSString *fullName3 = ((ATLUserMock*)set3.allObjects.firstObject).fullName;
+    expect(fullName3).toNot.equal([ATLUserMock userWithMockUserName:ATLMockUserNameKlemen].fullName);
+}
+
+
```
Comment (kcoleman731) [inline Tests/ATLUserMockTest.m]: extra line after closing brace

File: Tests/ATLUserMockTest.m [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#import "ATLTestInterface.h"
+#import "ATLSampleConversationViewController.h"
+
+@interface ATLUserMockTest : XCTestCase
+
+@end
+
+@implementation ATLUserMockTest
+
+- (void)setUp {
```
Comment (kcoleman731) [inline Tests/ATLUserMockTest.m]: we can remove setup and teardown methods is not being used.

**Comment by kmahal** [comment]
updated format fixes

File: Tests/ATLUserMockTest.m [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#import "ATLTestInterface.h"
+#import "ATLSampleConversationViewController.h"
+
+@interface ATLUserMockTest : XCTestCase
+
+@end
+
+@implementation ATLUserMockTest
+
+-(void)testToVerifyCaseInsensitiveSearch
```
Comment (kcoleman731) [inline Tests/ATLUserMockTest.m]: space in between - and (void)

**Pull request merged**